frappe/database/database.py

```diff
@@ -119,6 +119,9 @@ def sql(self, query, values=(), as_dict = 0, as_list = 0, formatted = 0,
 		if not run:
 			return query
 
+		# remove \n \t from start and end of query
+		query = re.sub(r'^\s*|\s*$', '', query)
```

Comment:
@shariquerik  @surajshetty3416 `str.strip()` is simpler and less prone to unwanted side effects of regex.

Also no tests? :smirk: